pkg/services/sqlstore/sqlstore.go

```diff
@@ -294,7 +294,7 @@ type ITestDB interface {
 func InitTestDB(t ITestDB) *SqlStore {
 	t.Helper()
 	sqlstore := &SqlStore{}
-	sqlstore.skipEnsureAdmin = true
+	sqlstore.skipEnsureAdmin = setting.DisableAdminUser
```

Comment:
This is only used for test and `skipEnsureAdmin` should always be false (as before). You want to read the `setting.DisableAdminUser` in the `Init()` method since that method is called when Grafana starts up.

Reply:
I just figured that out! LOL Thanks!